TsRandomizer/Randomisation/ItemLocationMap.cs

```diff
@@ -256,7 +256,7 @@ void AddPresentItemLocations()
 			Add(new ItemKey(2, 23, 136, 304), "Air Vents left", ItemProvider.Get(EInventoryRelicType.ElevatorKeycard), UpperRightSideLibrary & (R.CardE | R.DoubleJump)); //needs only UpperRightSideLibrary but requires Elevator Card | Double Jump to get out
 			Add(new ItemKey(2, 11, 104, 192), "Varndagroth right bottom floor", ItemProvider.Get(EInventoryUseItemType.EssenceCrystal), LowerRightSideLibrary);
 			Add(new ItemKey(2, 29, 280, 222 + TimespinnerSpindle.YOffset), "Varndagroth", ItemProvider.Get(EInventoryRelicType.TimespinnerSpindle), RightSideLibraryElevator & R.CardC);
-			Add(new RoomItemKey(2, 52), "Spider hell", ItemProvider.Get(EInventoryRelicType.TimespinnerGear2), RightSideLibraryElevator & R.CardA);
+			Add(new RoomItemKey(2, 52), "Varndagroth tower A door", ItemProvider.Get(EInventoryRelicType.TimespinnerGear2), RightSideLibraryElevator & R.CardA); //actually describes location, also avoids name "Spider hell 2"
```

Comment:
These are named Varndagroth Spider hell and Lab Spider hell in Archipelago and regular Timespinner adds the area name in the spoiler log so there isn't actually any confusion here